Automattic/jetpack
# Protect: Add WAF footer

#### Description
Adds a footer section to the Protect plugin WAF UI, as per designs.

#### Changes proposed in this Pull Request:

* Adds a `FirewallFooter` component and applies it to the `FirewallPage`
* Adds a `StandaloneModeModal` component to the `FirewallFooter`

#### To-Do:
- [ ] Make `CheckboxControl` component functional

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1203282049665702

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:





* Start Jurassic Tube instance using this Protect branch
* Load the Firewall page and verify the `FirewallFooter` component displays as expected (in both desktop and mobile and viewports in between)
* Click the Learn more and ensure the `StandaloneModeModal` displays as expected (in both desktop and mobile and viewports in between)
